src/core/Akka.Tests/Loggers/LogFilterEvaluatorSpecs.cs

```diff
@@ -227,4 +227,199 @@ public void ShouldFilterByLogMessageContains(LogEvent e, bool expected)
             Assert.Equal(expected, keepMessage);
         }
     }
+
+    /// <summary>
+    /// Tests that log filtering works correctly with semantic logging templates
+    /// </summary>
+    public class SemanticLoggingFilterCases
```

Comment:
Make sure log filtering functionality works with semantic logging - should help us move forward with https://github.com/akkadotnet/Akka.Logger.Serilog/pull/289 hopefully